readme.md

```diff
@@ -30,6 +30,10 @@ avaFiles.isSource(filePath);
 avaFiles.findTestFiles().then(files => {
 	// files is an array of found test files
 });
+
+avaFiles.findTestHelpers().then(files => {
+	// files is an array of found test helpers and fixtures
```

Comment:
From the point of this method's functionality of precompiling code, they're the same. Are you referring to the function name or comment?